test/Ported/Issues/RavenDB_11552.ts

```diff
@@ -92,7 +92,7 @@ describe("RavenDB-11552", function () {
             assertThat(company)
                 .isNull();
             assertThat(session.advanced.isLoaded("companies/1"))
-                .isFalse();
+                .isTrue();

```

Comment:
why is there a change in behavior?

Reply:
look here: https://github.com/ravendb/ravendb/commit/6a2214edb0d3074d26f461fdf199dd29118bcf79#diff-ec982d440e75ea9ecdf23c70ed54dcc9